README.md

```diff
@@ -40,12 +40,12 @@ The licenses for the applications themselves can be found in the LICENSE files.
 Older compiler versions might work but are not supported.
 
 ### Linux, macOS, FreeBSD
-  * GCC ≥ 7
-  * Clang/LLVM ≥ 9
-  * Intel Compiler ≥ 2021.3.0 (Intel OneAPI)
+  * GCC ≥ 11
```

Comment:
```suggestion
  * GCC ≥ 10
```

Reply:
Stimmt, habe den gcc10 in der CI übersehen.